static/translations/messages_en.json

```diff
@@ -231,5 +231,6 @@
   "Signed out": "Signed out",
   "Remove account?": "Remove account?",
   "This will make the mobile app on this device forget {email} on {realmUrl}.": "This will make the mobile app on this device forget {email} on {realmUrl}.",
-  "Remove account": "Remove account"
+  "Remove account": "Remove account",
+  "This message was hidden because it is from a user you have muted. Long-press to view.": "This message was hidden because it is from a user you have muted. Long-press to view."
```

Comment:
Very much a nit: I like to add entries in the middle here -- next to logical neighbors if any, but in any case somewhere other than the end. The reason is that that lets the diff avoid touching lines it doesn't have to.

Reply:
I've sometimes thought that it'd be nice to enforce alphabetical order of the entries. It's impossible (or would take way too much effort, anyway) to be rigorous about grouping together similar items in all cases, and alphabetizing would eliminate cases where it's hard to decide where something goes.

I think I wouldn't want to alphabetize unless we found a sensible way to automate it, that we could also count on contributors using too. That'd take a bit of work, I think, and it's possibly not worth it.

Reply:
We could just add new entries at the top, instead of the bottom :p

(I agree that making tooling or whatever for this is probably not worth it, and I moved it up a bit in this commit)